src/rust/graph-mutation/src/main.rs

```diff
@@ -0,0 +1,74 @@
+use std::{
+    net::SocketAddr,
+    sync::Arc,
+    time::Duration,
+};
+
+use clap::Parser;
+use graph_mutation::{
+    config::GraphMutationServiceConfig,
+    graph_mutation::GraphMutationManager,
+    reverse_edge_resolver::ReverseEdgeResolver,
+};
+use rust_proto::{
+    client_factory::build_grpc_client,
+    graplinc::grapl::api::graph_mutation::v1beta1::server::GraphMutationServer,
+    protocol::healthcheck::HealthcheckStatus,
+};
+use scylla::CachingSession;
+use tokio::net::TcpListener;
+use uid_allocator::client::CachingUidAllocatorServiceClient as CachingUidAllocatorClient;
+
+const SERVICE_NAME: &'static str = "graph-mutation";
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
Note: eyre::Result would be more helpful than Box dyn Error, it'll print error chains and the src line where the error occured. https://app.slack.com/client/T017B8DN5GB/C0174A8QV2S